README.md

```diff
@@ -6,6 +6,12 @@
 - [Sphinx](http://www.sphinx-doc.org/)
 - [Read the Docs Sphinx Theme](https://github.com/rtfd/sphinx_rtd_theme)
 
+### Debian/Ubuntu
+
+`sudo apt install python-sphinx`
+
+`pip install sphinx_rtd_theme`
```

Comment:
can we, instead of that, follow standard pip `requirements.txt` file?
then, we can even control version of packages.
eg
```
Sphinx==1.6.5
sphinx_rtd_theme==0.3.0
```

Reply:
I added `requirements.txt` as suggested, so the README.md can be changed to suggest running `pip install -r requirements.txt` and we don't need platform-specific instructions.

Reply:
@keradus From my side if perfect also like this. I encountered the issue when trying to install on my Ubuntu machine and while searching the Sphinx online I only found the apt install way.